liquidprompt

```diff
@@ -584,6 +584,7 @@ fi
 # Where are we? #
 #################
 
+if ! type _lp_connection &> /dev/null; then
```

Comment:
we could add a check like `! type _mock_lp_connection` and then make `_lp_connection` call that. But that would introduce test specific code...